### PR TITLE
MB-8107: Adds helper text to Customer home screen when user has unapproved amended orders

### PR DESCRIPTION
**Pull request by kylehilltruss**
## Description

Design requested that a helper be added to the Customer home page, which displays once a user has uploaded amended orders documents until the TOO has approved the amendments. This PR adds that helper and the conditional logic to display it.

There is a storybook entry added for the helper, and also the state within in the Customer home page that would display this helper.

## Reviewer Notes

Ensure that the added test cases are useful -- the Home page component's tests are written with enzyme, which I haven't used in quite a while. (It didn't seem worth the effort to convert it to RTL at this time.)

## Setup

Right now, this change can only be seen in the story added to storybook (`Customer Components` / `Pages` / `Home`). 

Once the backend-focused branch *mb-8476-customer-has-a-save-button-for-uploading-amended-orders* is merged into master, you'll be able to upload and save amended orders as a Customer with a submitted move and see the helper message display on the home page. 

```sh
make db_dev_e2e_populate
make client_run
make server_run
```

(For the sake of clarity, this PR can be merged safely as-is; it will just have no practical effect until the complementary branch is merged.)

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* Closes [MB-8107](https://dp3.atlassian.net/browse/MB-8107).
* Requires [MB-8476](https://dp3.atlassian.net/browse/MB-8476) to be completed before this will practically work.

## Screenshots

There's not really a "before"; [here's the after](https://user-images.githubusercontent.com/83614364/124020251-08378c00-d9b8-11eb-9dfc-1a6ba28435e7.png).

### CODE REVIEW

File: src/pages/MyMove/Home/index.test.jsx [hunk 1]
```diff
@@ -305,5 +306,35 @@ describe('Home component', () => {
         expect(wrapper.find('HelperSubmittedPPM').exists()).toBe(true);
       });
     });
+
+    describe('for amended orders', () => {
```
Comment (YanZ777) [inline src/pages/MyMove/Home/index.test.jsx:310]: You can mix RTL and enzyme in the same file, so if you want to make this RTL, feel free to do so.

File: src/pages/MyMove/Home/index.test.jsx [hunk 2]
```diff
@@ -305,5 +306,35 @@ describe('Home component', () => {
...
+            uploadedOrderDocuments={uploadedOrderDocuments}
+            uploadedAmendedOrderDocuments={uploadedAmendedOrderDocuments}
+            move={move}
+            currentPpm={currentPpm}
+          />
+        </MockProviders>,
+      );
+
+      it('renders the HelperAmendedOrders helper', () => {
+        expect(wrapper.find('HelperAmendedOrders').exists()).toBe(true);
```
Comment (YanZ777) [inline src/pages/MyMove/Home/index.test.jsx:336]: Should there also be a test to check that it doesn't render the Helper based on if there are amended orders or the state?

Comment (kylehilltruss) [reply src/pages/MyMove/Home/index.test.jsx:336]: There is, I just forgot to push up the commit containing it last evening 🙃

**Review by YanZ777 — APPROVED** [review]
Looks good to me (and implicitly Marjo and Felipe since they looked at it with me).

**Review by Shkeating — APPROVED** [review]

**Pull request merged**